app/views/tree/_dep.html.erb

```diff
@@ -9,7 +9,7 @@
   <em class='text-muted'>
     <span class="tip" title="Specified version range"><%= requirements %></span>
     <%= '-' if requirements.present? %>
-    <%= project.normalize_licenses.join(', ') %>
+    <%= project.normalized_licenses.join(', ') %>
```

Comment:
@katzj @havocp do you know the background on `normalized_licenses` vs `license_normalized`?

Reply:
(the latter is set by `normalize_licenses`, so I'm confused by the singular/plural diff there)